src/MagnumPlugins/AssimpImporter/Test/skin.dae

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The new files should get referenced in the test CMakeLists again :)

Reply:
Whoops, putting this on my todo list so I can do this towards the end in case more test files are needed